content/blog/2021-03-31-march-21-community-gems.md

```diff
@@ -0,0 +1,128 @@
+---
+title: March '21 Community Gems
+date: 2021-03-31
+description: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, hyperparameter tuning,
+  best practices for managing experiments and more.
+picture: 2021-03-31/gems-cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/march-2021-community-gems/708
+tags:
+  - Discord
+  - Gems
+  - Pipelines
+  - Remote Storage
+---
+
+### [Q: Will DVC work with <my remote cloud storage of choice?>](https://discord.com/channels/485586884165107732/563406153334128681/821493606770409493)
+
+We recently had questions about this, specifically regarding Huawei Cloud and
+Backblaze B2 Storage. The answer is any cloud storage that has an S3 interface
+will work with DVC and both of the aforementioned do! In addition DVC works with
+Azure, Google Drive, GS, OSS and SSH.
```

Comment:
💅  should it be `, and SSH`?

Reply:
It's a matter of taste.  Both are accepted in punctuation.  I always leave out the extras.  But can add if want.  Do we use the comma before and everywhere else?

Reply:
> Both are accepted in punctuation.

curious - because in this case it doesn't change the meaning?

no strong opinions here - feel free to resolve when you see it's ready

Reply:
I'm just going to put it in case OSS and SSH can be one thing in addition to them being separate since I have know idea what they are 😏